ush/python/pygfs/jedi/jedi.py

```diff
@@ -216,18 +218,45 @@ def get_bias_dict(self, task_config: AttrDict) -> Dict[str, Any]:
                 obfile = ob['obs bias']['input file']
                 obdir = os.path.dirname(obfile)
                 basename = os.path.basename(obfile)
-                prefix = '.'.join(basename.split('.')[:-2])
-                for file in ['satbias.nc', 'satbias_cov.nc', 'tlapse.txt']:
-                    bfile = f"{prefix}.{file}"
-                    copylist.append([os.path.join(task_config.COM_ATMOS_ANALYSIS_PREV, bfile), os.path.join(obdir, bfile)])
-                    # TODO: Why is this specific to ATMOS?
+                prefix = '.'.join(basename.split('.')[:-3])
+                bfile = f"{prefix}.{bias_file}"
+                tar_file = os.path.join(obdir, bfile)
+                copylist.append([os.path.join(task_config.VarBcDir, bfile), tar_file])
+                break
 
         bias_dict = {
             'mkdir': [os.path.join(task_config.DATA, 'bc')],
             'copy': copylist
         }
+
         return bias_dict
 
+    @logit(logger)
+    def extract_tar(self, task_config: AttrDict, tar_dict) -> Dict[str, Any]:
+        """Extract files from list of tarfiles
+
+        This method extract bias correction files from tarball(s)
+
+        Parameters
+        ----------
+        task_config: AttrDict
+            Attribute-dictionary of all configuration variables associated with a GDAS task.
+        tar_dict
+            a dictionary containing the list of tar files
+
+        Returns
+        ----------
+        None
+        """
+
+        # extract bias correction files from tar file
+        for tar_file in tar_dict['copy']:
+            if ".tar" in tar_file[1]:
+                with tarfile.open(tar_file[1], "r") as tarball:
+                    tarball.extractall(path=os.path.join(task_config.DATA, 'obs'))
+                    logger.info(f"Extract {tarball.getnames()}")
+        tarball.close()
```

Comment:
I don't think this is needed because of `with tarfile.open` implies it closes when it's done. Plus, if we do need it, shouldn't it be in the for loop? Seems out of place here and surprised it isn't failing.

Reply:
I can remove `tarball.close()`.  I included it because I found it in some [python 3 tarfile examples](https://docs.python.org/3/library/tarfile.html#tar-examples).

Reply:
Part of the magic of context managers (what `with` does) is it automatically opens and closes the file as part of the process. See https://realpython.com/python-with-statement/

Reply:
`tarball.close()` removed as part of [0ab7bdd](https://github.com/NOAA-EMC/global-workflow/pull/2875/commits/0ab7bdd6083051f5cb9b9bc687d2a4bbbe5e2e34)

Reply:
Resolving since the line in question has been removed.